core/server/master/src/main/java/alluxio/master/file/contexts/OperationContext.java

```diff
@@ -22,14 +22,31 @@
 public class OperationContext<T extends com.google.protobuf.GeneratedMessageV3.Builder<?>> {
   // Proto message that is being wrapped
   private T mOptionsBuilder;
+  // Used to track client call status.
+  private CallTracker mCallTracker;
```

Comment:
when will this be `null` and can this be `final`?

Reply:
In order to reduce amount of code change, I introduced another constructor here that sets the tracker. I can not make it final because of that.

It will be null, when contexts for calls that requires tracking is created with constructor that doesn't take tracker. (E.g unit tests, where we don't have a tracking facility).

Now, I created another static tracker for no-op tracker that always return false, and pass that instead of null. So this check is gone.